docs/src/Fieldset.doc.js

```diff
@@ -0,0 +1,222 @@
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+      function RadioButtonExample() {
+        const [favorite, setFavorite] = React.useState(undefined);
+        const [name, setName] = React.useState('')
+        const [email, setEmail] = React.useState('')
+
+        return (
+          <form>
+            <Flex direction="column" gap={4}>
+              <Text>Profile Details</Text>
+              <TextField
+                id="name"
+                onChange={({ value }) => setName(value)}
+                placeholder="First and last name"
+                label="Name"
+                value={name}
+                type="email"
+              />
+              <TextField
+                id="email"
+                onChange={({ value }) => setEmail(value)}
+                placeholder="example@test.com"
+                label="Email"
+                value={email}
+                type="email"
+              />
+              <Fieldset legend="What is your favorite pet?">
+                <Flex direction="column" gap={2}>
+                  <RadioButton
+                    checked={favorite === 'dogs'}
+                    id="favoriteDogA11y"
+                    label="Dogs"
+                    name="favorite"
+                    onChange={() => setFavorite( 'dogs' )}
+                    value="dogs"
+                  />
+                  <RadioButton
+                    checked={favorite === 'cats'}
+                    id="favoriteCatA11y"
+                    label="Cats"
+                    name="favorite"
+                    onChange={() => setFavorite( 'cats' )}
+                    value="cats"
+                  />
+                  <RadioButton
+                    checked={favorite === 'plants'}
+                    id="favoritePlantsA11y"
+                    label="Plants"
+                    name="favorite"
+                    onChange={() => setFavorite( 'plants' )}
+                    value="plants"
+                  />
+                </Flex>
+              </Fieldset>
+            </Flex>
+          </form>
+        );
+      }
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(<MainSection name="Localization" description={`Be sure to localize the \`legend\` text.`} />);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      description={`Be default, the \`legend\` is visible above the items in the Fieldset. However, if the form items are labelled by content elsewhere on the page, or a more complex legend is needed, the \`legendDisplay\` prop can be used to visually hide the legend. In this case, it is still available to screen reader users, but will not appear visually on the screen.`}
```

Comment:
By default typo

Reply:
not strong about this, but maybe we can specify what the screen readers will read something like.

In the example below,  screenreaders will read in the following order:

1. Design options
2. Choose which backgrounds to use as placeholders when creating content
3. Which backgrounds would you like to use?
4.